styles.css

```diff
@@ -96,53 +100,51 @@
 
 .bottom_geometry {
   position: absolute;
-  width: 375px;
+  width: 380px;
   height: 160px;
   top: 492px;
   background-image: url("src/img/svg/image_geometry_2.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  align-self: center;
 }
 
 .colaboration_button_rectangle:hover:enabled {
-  position: absolute;
-  width: 288px;
-  height: 50px;
-  left: 16px;
-  top: 390px;
   border: 1px solid #1a2236;
   box-sizing: border-box;
   background-color: hsl(0, 0%, 100%);
   color: #1a2236;
 }
 
 .colaboration_button_rectangle:active:enabled {
-  position: absolute;
-  width: 288px;
-  height: 50px;
-  left: 16px;
-  top: 390px;
   border: 2px solid #1a2236;
   box-sizing: border-box;
   color: #1a2236;
   background-color: hsl(0, 0%, 100%);
 }
 
 .about_section {
-  position: absolute;
+  position: relative;
   display: flex;
```

Comment:
- [ ] instead of using comments like `You can re-use most of the CSS styling, this will make your .css line reduce in number` you can be more polite by saying (in order to keep code clean and DRY kindly put `display: flex` in a class and add it to your HTML element when needed)